examples/nextjs-app-router/components/Form.tsx

```diff
@@ -102,7 +123,7 @@ export function Input(inputProps: ComponentProps<"input"> & { name: string }) {
 	return (
 		<>
 			<input
-				className={twMerge("bg-gray-800", inputProps.className)}
+				className={twMerge("", inputProps.className)}
```

Comment:
```suggestion
				className={twMerge("dark:bg-gray-800", inputProps.className)}
```
I assume you reverted because you're on light mode and couldn't see the input. I was on dark mode 🙃 This should use a dark background on dark mode only